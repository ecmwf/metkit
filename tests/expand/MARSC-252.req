retrieve,
    class=od,
    stream=da,
    type=an,
    levtype=sfc,
    levelist=off,
    resol=off,
    date=20250416,
    time=00,
    step=00,
    param=tcc/10u/10v/2t/msl,
    param=tcc,
    repres=gg,
    format=p,
    expver=0001,
    target=data.reference 