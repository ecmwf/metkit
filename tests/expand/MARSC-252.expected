RETRIEVE,
    CLASS      = OD,
    TYPE       = AN,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = GG,
    LEVTYPE    = SFC,
    PARAM      = 164,
    DATE       = 20250416,
    TIME       = 0000,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = data.reference 